src/main/resources/assets/plethora/lang/nl_be.lang

```diff
@@ -0,0 +1,148 @@
+# Modules
+item.plethora.module.module_introspection.name=Introspectie-module
+item.plethora.module.module_introspection.adjective=Introspectieve
+item.plethora.module.module_introspection.desc=Verpakt je inventaris als randapparatuur. Shift+Gebruik om te linken met speler.
+
+item.plethora.module.module_laser.name=Frickin' laserstraal
+item.plethora.module.module_laser.adjective=Schietend
+item.plethora.module.module_laser.desc=Het schiet lasers. Genoeg gezegd.
+
+death.attack.laser=%1$s was verdampt door %2$s
+death.attack.laser.item=%1$s was verdampt by %2$s met %3$s
```

Comment:
"by" hasn't been translated to Dutch with "door" like the previous line.

Reply:
Already removed in aba8ec842f78f519d50b7fedf94c1c09f257f9ed